### PR TITLE
add tests for a vpn subscribed user sign in and out

### PR DESCRIPTION
**Pull request by ashrivastava-qa**
adding tests for signing in and out for a VPN subscribed user account.

### CODE REVIEW

**Comment by ashrivastava-qa** [comment]
@isabelrios @rlr I know that the new IOS tests needs to be added for iOS VPN version 2.0 (cross-platform) but since we already started working on this test, I created a PR for this.

File: FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift [hunk 1]
```diff
@@ -113,4 +118,57 @@ class FirefoxPrivateNetworkVPNUITests: BaseTestCase {
         waitForExistence(app.staticTexts["Mozilla VPN"], timeout: 15)
         XCTAssertTrue(app.staticTexts["Mozilla VPN"].exists, "The main page is not loaded correctly")
         }
-    }
+
+    func testSignInAsSubscribedUser() {
+            // The main screen is shown
```
Comment (isabelrios) [inline FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift:123]: Nit: there is extra space for this and following lines. Look at previous test to get them equally aligned.

File: FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift [hunk 2]
```diff
@@ -113,4 +118,57 @@ class FirefoxPrivateNetworkVPNUITests: BaseTestCase {
...
+            waitForExistence(app.staticTexts["Mozilla VPN"], timeout: 15)
+            XCTAssertTrue(app.staticTexts["Mozilla VPN"].exists, "The main page is not loaded correctly")
+
+            // Tap on Get started
+            app.buttons["Get started"].tap()
+            waitForExistence(app.staticTexts["Use a different account"], timeout: 15)
```
Comment (isabelrios) [inline FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift:129]: I would add a comment here // Removing previous account or something, you are saying Tap on Get started, but then what follows is not very clear

**Review by isabelrios — APPROVED** [review]
LGTM, only nit commets to fix before merging and don't forget to squash your commits and/or choose the correct option for the Github merge button (Squash and Merge)

**Pull request merged**